app/controllers/api/v1/users/omniauth_callbacks_controller.rb

```diff
@@ -23,12 +18,17 @@ def auth
           @auth ||= request.env['omniauth.auth']
         end
 
-        def render_success
-          render json: { success: true }, status: :ok
-        end
+        def render_response(user)
+          oauth_token = OauthToken.generate_access_token(user)
+          data = OauthTokenSerializer.new(oauth_token).serializable_hash[:data]
+
+          message = if user.persisted?
```

Comment:
Do we need this message? As this is an API, so I think this message is redundant. We often use it to show in the view - [flash message](https://www.rubyguides.com/2019/11/rails-flash-messages/#:~:text=A%20flash%20message%20is%20a,User%20not%20found%E2%80%9D%20(error))

Reply:
@hoangmirs I added the message to determine the case user sign in or sign up 😄 Would it be possible to keep it here? 😅

Reply:
It is a security concern. Hackers could use the API message to check if a user account exists. It is best to remove any such information.